firefox-ios/Client/Frontend/Browser/DownloadHelper/MIMEType.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UniformTypeIdentifiers
+
+struct MIMEType {
+    static let Bitmap = "image/bmp"
+    static let CSS = "text/css"
+    static let GIF = "image/gif"
+    static let JavaScript = "text/javascript"
+    static let JPEG = "image/jpeg"
+    static let HTML = "text/html"
+    static let OctetStream = "application/octet-stream"
+    static let Passbook = "application/vnd.apple.pkpass"
+    static let PDF = "application/pdf"
+    static let PlainText = "text/plain"
+    static let PNG = "image/png"
+    static let WebP = "image/webp"
+    static let Calendar = "text/calendar"
+    static let USDZ = "model/vnd.usdz+zip"
+    static let Reality = "model/vnd.reality"
+    static let OpenDocument = "application/msword"
+    static let MicrosoftWord = "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
```

Comment:
Q: Aren't the last two doc types actually the other way around?

Reply:
that's what I thought initially but when I added `application/msword` and tested it allowed the download of `odt` extension file and for `docx` the MIMEType was "application/vnd.openxmlformats-officedocument.wordprocessingml.document".

Reply:
Ah, okay! It looked like DOC vs DOCX to me at first, with ODT having the `"vnd.oasis.opendocument.*"` bit as listed e.g. at https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types but good it's actually working this way. (It always depends on the host/OS handling and quirks thereof, sorry for bothering with something already verified empirically;)…)